app/views/casework/crime_applications/_income_details.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= render partial: 'casework/crime_applications/sections/employment', locals: { income_details: crime_application.means_details.income_details } %>
 <%= render partial: 'casework/crime_applications/sections/income', locals: { income_details: crime_application.means_details.income_details } %>
 <%= render partial: 'casework/crime_applications/sections/income_payments', locals: { income_payments: crime_application.income_payments.formatted_income_payments } %>
+<%= render partial: 'casework/crime_applications/sections/income_benefits', locals: { income_benefits: crime_application.income_benefits.formatted_income_benefits } %>
```

Comment:
Income benefits was displaying at the bottom of the income section so corrected to displaying below payments